pvlib/iotools/psm3.py

```diff
@@ -10,8 +10,9 @@
 from json import JSONDecodeError
 
 NSRDB_API_BASE = "https://developer.nrel.gov"
-PSM_URL = NSRDB_API_BASE + "/api/solar/nsrdb_psm3_download.csv"
-TMY_URL = NSRDB_API_BASE + "/api/nsrdb_api/solar/nsrdb_psm3_tmy_download.csv"
+PSM_URL = NSRDB_API_BASE + "/api/nsrdb/v2/solar/psm3-download.csv"
+TMY_URL = NSRDB_API_BASE + "/api/nsrdb/v2/solar/psm3-tmy-download.csv"
+PSM5MIN_URL = NSRDB_API_BASE + "/api/nsrdb/v2/solar/psm3-5min-download.csv"
 
 # 'relative_humidity', 'total_precipitable_water' are not available
 ATTRIBUTES = [
```

Comment:
I have a mild preference for converting this to a tuple so that the default argument is not mutable.